downloads/swan-lake-release-notes/swan-lake-2201.1.0/RELEASE_NOTE.md

```diff
@@ -0,0 +1,688 @@
+- Fixed incorrect caller type name validation in the gRPC compiler plugin
+- Fixed passing Protobuf predefined types as repeated values and values in messages
+
+To view bug fixes, see the [GitHub milestone for Swan Lake 2201.1.0](https://github.com/ballerina-platform/ballerina-standard-library/issues?q=is%3Aclosed+is%3Aissue+milestone%3A%222201.1.0%22+label%3AType%2FBug).
+
+## Deployment updates
+
+### New features
+
+Added the `name` field for the `cloud.config.files` property in the `Cloud.toml` file to change the name of the generated config map 
+
+### Improvements
+- Reduced the package size of `ballerina/cloud`
+- Docker image generation now relies on the user's docker client
+- Added the `ballerinax/awslambda` package in [Ballerina Central](https://central.ballerina.io/ballerinax/awslambda)
+- Added the `ballerinax/azure_functions` package in [Ballerina Central](https://central.ballerina.io/ballerinax/azure.functions)
+
+### Breaking changes
+
+For existing `ballerinax/awslambda` and `ballerinax/azure_functions` projects, change the version to `2.1.0` in the `Dependencies.toml` file.
+
+### Bug fixes
+
+To view bug fixes, see the [GitHub milestone for Swan Lake 2201.1.0](https://github.com/ballerina-platform/module-ballerina-c2c/issues?q=is%3Aissue+is%3Aclosed+label%3AType%2FBug+milestone%3A%22Ballerina+Swan+Lake+-+2201.1.0%22).
+
+## Developer tools updates
+
+### New features
+
+#### AsyncAPI tool
+
+Introduced the Ballerina AsyncAPI tool, which will make it easy for you to start the development of an event API documented in an AsyncAPI contract in Ballerina by generating Ballerina service and listener skeletons. Ballerina Swan Lake supports the AsyncAPI Specification version 2.x. For more information, see [Ballerina AsyncAPI support](/learn/ballerina-asyncapi-support) and [AsyncAPI CLI documentation](/learn/cli-documentation/asyncapi/#asyncapi-to-ballerina).
+
+#### GraphQL tool
+
+Introduced the Ballerina GraphQL tool, which will make it easy for you to generate a client in Ballerina given the GraphQL schema (SDL) and GraphQL queries. Ballerina Swan Lake supports the GraphQL specification [October 2021 edition](https://spec.graphql.org/October2021/). For more information, see [Ballerina GraphQL support](/learn/ballerina-graphql-support/) and [GraphQL CLI documentation](/learn/cli-documentation/graphql/#graphql-to-ballerina).
+
+#### Language server
+
+- Added completion and code action support for already-imported modules in the Ballerina user home
+- Implemented file operation events in the Language Server
+
+### Improvements
+
+#### Debugger
+
+Added runtime breakpoint verification support. With this improvement, the debugger is expected to verify all the valid breakpoint locations in the current debug source. All the breakpoints that are set on non-executable lines of code (i.e., Ballerina line comments, documentation , blank lines, declarations, etc.) will be marked as `unverified` in the editor.
```

Comment:
```suggestion
- Added runtime breakpoint verification support. With this improvement, the debugger is expected to verify all the valid breakpoint locations in the current debug source. All the breakpoints that are set on non-executable lines of code (i.e., Ballerina line comments, documentation , blank lines, declarations, etc.) will be marked as `unverified` in the editor.
```

Reply:
We don't need a bullet point when there is only one single paragraph (which is not a list).

Reply:
@praneesha Ack. +1